README.rst

```diff
@@ -3,6 +3,61 @@ django-pldp
 
 Reusable Django app that implements the structure implied by the PDLP.
 
+Setup
+-----
+
+1. Install `django-pldp` in the Django app where you'd like to use it:
+
+`git submodule add git@github.com:datamade/django-pldp.git`
+
+2. Install dependencies and add them to your `requirements.txt` file:
+
```

Comment:
Alternatively, we can have a `requirements.txt` file inside the `pldp` app in this repo. That might be a little clearer organizationally.